roles/ceph-common/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: install dependencies
+  apt: pkg={{ item }}
+       state=present
+       update_cache=yes
+       cache_valid_time=3600
+  with_items: ceph_common.debian_pkgs
+
+- name: install ceph
+  apt: pkg={{ item }}
+       state=present
+  with_items: ceph_common.ceph_pkgs
+
+- name: check for a ceph socket
+  shell: "stat /var/run/ceph/*.asok > /dev/null 2>&1"
+  changed_when: false
+  failed_when: false
+  register: socket
+
+- name: wait for ceph directory
+  wait_for: path=/etc/ceph/
```

Comment:
I amended this. Seems it was moving on to the next task before the `/etc/ceph/` was created.